GVFS/GVFS.UnitTests/Maintenance/PackfileHelper.cs

```diff
@@ -0,0 +1,152 @@
+﻿using GVFS.Common;
+using GVFS.Common.FileSystem;
+using GVFS.Common.Git;
+using GVFS.Common.Tracing;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace GVFS.UnitTests.Maintenance
+{
+    public class PackfileHelper

```

Comment:
Did you intend to duplicate this class? I don't see it being used anywhere.

Reply:
I did not! I created it in the wrong place and cross-project move instead was a copy. Thanks for catching.